catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -151,53 +152,23 @@ public void testFullThread() throws Exception {
         List<Metacard> metacards = cswRecords.getCswRecords();
 
         assertThat(metacards, not(nullValue()));
```

Comment:
Should be able to replace the four lines with `assertThat(metacards, contains(metacard));`.